# Zadanie 1

Zmierzono czas pracy (w godzinach) piętnastu wylosowanych bateryjek radiowych i
otrzymano następujące wyniki: 35,34; 36,26; 30,54; 38,2; 37,59; 39,18; 33,16; 34,23;
27,9; 36,33; 32,39; 34,89; 35,7; 31,99; 34,03.
Zakładając, że czas pracy bateryjek ma rozkład normalny sprawdź na poziomie istotności 0,05
czy przeciętny czas pracy jest równy 35?


## 1. Sformułowanie hipotez


H0 (μ = 35): przeciętny czas pracy bateryjki wynosi 35h \
H1 (μ ≠ 35): przeciętny czas pracy bateryjki nie wynosi 35h

## 2. Wybór testu statystycznego
- Test parametryczny - wartość przeciętna
- Pojedyncza próba
- Skala ilorazowa

-> Test T

## 3. Poziom istotności  
α=0,05


## 4. Obliczenie statystyki testowej i p-value oraz podjęcie decyzji weryfikacyjnej

In [1]:
czas <- c(35.34, 36.26, 30.54, 38.2, 37.59, 39.18, 33.16, 34.23, 27.9, 36.33, 32.39, 34.89, 35.7, 31.99, 34.03)
t.test(czas, mu=35)


	One Sample t-test

data:  czas
t = -0.62579, df = 14, p-value = 0.5415
alternative hypothesis: true mean is not equal to 35
95 percent confidence interval:
 32.85422 36.17644
sample estimates:
mean of x 
 34.51533 


Z otrzymanego wydruku możemy odczytać m.in.  wartość statystyki testowej (t = -0.62579) oraz  p-wartość = 0.5415. \
Ponieważ: p>α, zatem nie ma podstaw do odrzucenia hipotezy zerowej.


## 5. Wnioski:
Oznacza to, że średni czas pracy bateryjski nie różni się istotnie od 35h.

Możliwe było popełnienie błędu II rodzaju polegającego na nieodrzuceniu hipotezy zerowej, chociaż jest fałszywa.


# Zadanie 2

W celu zmniejszenia zużycia paliwa Komisja Doradcza Przewozów Morskich zleciła
kompaniom dalekomorskim zredukowanie prędkości statków do 17 węzłów. Jedna z
kompanii, której statki pływają z reguły z prędkością od 18,5 do 19,8 węzłów przy dziennym
średnim zużyciu paliwa w wysokości 127,7 t, sprawdziła rejestr rejsów i odnalazła 5 takich
rejsów, dla których średnia prędkość wynosiła zaledwie 17 węzłów. Dzienne zużycie paliwa w
czasie tych rejsów wynosiło: 101,1; 105,7; 102,6; 113,4; 98,1 t. Zakładając, że dzienne
zużycie paliwa ma rozkład normalny, czy dane te potwierdzają słuszność Komisji, że obniżenie
prędkości do 17 węzłów zmniejszy zużycie paliwa poniżej 127,7, przy poziomie istotności
0,01?


## 1. Sformułowanie hipotez


H0: (μ >= 127.7 ): zmniejszenie prędkości nie bedzie miało wpływu na zużycie paliwa. \
H1: (μ < 127.7): zmiejszenie prędkości będzie miało wpływ na zmiejszenie zużycia paliwa


## 2. Wybór testu statystycznego
- Test parametryczny - wartość przeciętna
- Pojedyncza próba
- Skala ilorazowa

-> Test T

## 3. Poziom istotności  
α=0,01


## 4. Obliczenie statystyki testowej i p-value oraz podjęcie decyzji weryfikacyjnej

In [ ]:
paliwo <- c(101.1, 105.7, 102.6, 113.4, 98.1)
t.test(paliwo, mu=127.7, alternative = "less")


	One Sample t-test

data:  paliwo
t = -9.0106, df = 4, p-value = 0.00042
alternative hypothesis: true mean is less than 127.7
95 percent confidence interval:
     -Inf 109.7447
sample estimates:
mean of x 
   104.18 


Z otrzymanego wydruku możemy odczytać m.in.  wartość statystyki testowej (t = -9.0106) oraz  p-wartość = 0.00084. \
Ponieważ: p<α, zatem należy odrzucić hipotezę zerową.

## 5. Wnioski:
Porównując p-wartość z przyjętym poziomem istotności
α = 0,01 (p < α) stwierdzamy wyraźne postawy do odrzucenia H0, co oznacza, że zaproponowana zmiana zmmniejszenia prękości do 17 węzłów, jest słuszna.  Oznacza to, że średnie zużycie paliwa różni się (jest mniejsze) od 127.7 t.

Możliwe było popełnienie błędu I rodzaju, polegającego na odrzuceniu hipotezy zerowej, chociaż jest ona prawdziwa.


# Zadanie 3

Wytrzymałość na ciśnienie wewnętrzne jest ważną charakterystyką jakościową szkła butelek.
Pewna rozlewnia chce zamówić butelki, których średnia wytrzymałość przewyższa 1.20 N/mm2
.
Pobrano próbę losową 20 butelek, które następnie umieszczono w maszynie hydrostatycznej,
zwiększając ciśnienie aż do zniszczenia butelki i otrzymano następujące wyniki (w N/mm2
):
1.36, 1.14, 1.27, 1.15, 1.20, 1.29, 1.27, 1.18, 1.23, 1.36, 1.38, 1.37, 1.30, 1.21, 1.33, 1.28, 1.32,
1.29, 1.33, 1.25.
Zbadaj, czy na poziomie istotności równym 0,05 można przyjąć założenie o jego normalności
rozkładu?

## 1. Sformułowanie hipotez

H0: można przyjąć założenie o normalności rozkładu, przy założeniu poziomu istoności α=0,05

H1: nie można przyjąć założenia o normalności rozkładu, przy założeniu poziomu istoności α=0,05

## 2. Wybór testu statystycznego
- Test nieparametryczny - infomracja o normalności rozkładu
- Pojedyncza próba
- Skala ilorazowa

-> Test Shapiro-Wilka

## 3. Poziom istoności
α=0,05

## 4. Obliczenie statystyki testowej i p-value oraz podjęcie decyzji weryfikacyjnej

In [ ]:
wytzymalosc <- c(1.36, 1.14, 1.27, 1.15, 1.20, 1.29, 1.27, 1.18, 1.23, 1.36, 1.38, 1.37, 1.30, 1.21, 1.33, 1.28, 1.32, 1.29, 1.33, 1.25)
shapiro.test (wytzymalosc)


	Shapiro-Wilk normality test

data:  wytzymalosc
W = 0.95459, p-value = 0.4421


Z otrzymanego wydruku możemy odczytać m.in. wartość statystyki testowej (W = 0.95459) oraz p-wartość = 0.4421. \
Ponieważ: p>α, zatem nie ma podstaw do odrzucenia hipotezy zerowej.

## 5. Wnioski:

Oznacza to, że można przyjąć założenie o normalności rozkładu, przy założeniu poziomu istoności α=0,05

Możliwe było popełnienie błędu II rodzaju polegającego na nieodrzuceniu hipotezy zerowej, chociaż jest fałszywa.

# Zadanie 4



Dane przedstawiają emisję zanieczyszczeń powietrza pyłami polimerów (w t) pochodzącymi z kilku zakładów szczególnie uciążliwych dla wybranego województwa: w roku I oraz 5 lat później, w roku II.

Rok I 220 185 270 285 200 295 255 190 225 230  
Rok II 190 175 215 260 215 195 260 150 155 175

Na poziomie istotności 0,05 należy zweryfikować hipotezę, że nastąpił istotny spadek emisji
pyłów w tych zakładach.

## 1. Sformułowanie hipotez

H0: (μ_1=μ_2): nie nastąpiła różnica (istotny spadek) emisji pyłów w zakładach

H1: (μ_1<μ_2): nastąpiła różnica (istotny spadek) emisji płynów w zakładach


## 2. Wybór testu statystycznego
- Test parametryczny - wartość przeciętna
- Podwójna próba
- Skala ilorazowa

-> Test T

## 3. Poziom istotności  
α=0,05


## 4. Obliczenie statystyki testowej i p-value oraz podjęcie decyzji weryfikacyjnej

In [ ]:
przed <- c(220, 185, 270, 285, 200, 295, 255, 190, 225, 230)
po <- c(190, 175, 215, 260, 215, 195, 260, 150, 155, 175)

t.test(przed, po, alternative="greater", paired=TRUE)


	Paired t-test

data:  przed and po
t = 3.2863, df = 9, p-value = 0.004717
alternative hypothesis: true mean difference is greater than 0
95 percent confidence interval:
 16.13997      Inf
sample estimates:
mean difference 
           36.5 


Z otrzymanego wydruku możemy odczytać m.in.  wartość statystyki testowej (t = 3.2863) oraz  p-wartość = 0.004717 \
Ponieważ: p<α, zatem są podstawy do odrzucenia hipotezy zerowej.

## 5. Wnioski:
Porównując p-wartość z przyjętym poziomem istotności
α = 0,05 (p > α) stwierdzamy podstawy do odrzucenia H0, co oznacza, że nastąpił spadek emisji płynów w badanych zakładach.

Możliwe było popelnienie błędu I rzędu polegającego na odrzuceniu hipotezy zerowej, chociaż jest ona prawdziwa.
